blog/2023-07-18-wing-cloud-launch.md

```diff
@@ -0,0 +1,284 @@
+
+The last piece of our story is the **Wing Console**. It's a UI tool for interacting with cloud
+applications. It displays a visual diagram of your application which shows the hierarchical
+structure of your system as well as the relationships between resources. In the future, it will show
+Open Telemetry events as the system is activated.
+
+<!--![screenshot]()-->
+
+In the console, each resource also has a dedicated UI which can be used to interact with the
+resource. You can invoke functions, push messages to the queue, download and upload files to a
+bucket and even open a website. In the future, custom UIs can be created for any resource.
+
+Today, Wing Console only supports local development. It is baked into the Wing CLI and VSCode
+experience. To open it, just run `wing it` and it will just work with no additional setup needed. It
+will also watch for any file changes, recompile and reload as needed. Batteries included.
+
+**We are planning to offer Wing Console for production**. This means that users will be able to
+connect to a deployed system and operate their production systems through Wing Console. 
+
+<!--[Join the waiting list]()-->
+
+We believe this addresses a major pain when operating cloud applications. Today, we manage
+applications "from the bottom", i.e. from the resources up. I go to my cloud provider's
+administration console and what I see is a bunch of low-level resources - queues or buckets or
+functions. I don't see my application. I oftentimes also have to juggle to separate administrative
+tools for services that are outside of my provider's platform.
+
+Going back to my favorite Lego analogy: Think about a cloud application as this beautiful Lego
+castle. I've finished building my castle, and now I want to open the main gate. I log into the
+provider's console and I see a menu of Lego block types: all the reds, all the 4 piece blues, all
+the flat surfaces, etc. My gate is built out of reds and blues and greens and a bunch of cables. But
+I can't see a gate. I just see the building blocks from which it's constructed. How would I know
+which blue piece to move and where in order to open the gate?
+
+This is the experience we have today when we operate applications in the cloud. We get an alarm that
+there are some errors from some Lambda function. How do we know which part of the system is
+affected? Which team owns this part?
+
+The solution, today, is dashboards. I have to manually craft these views to represent the mental
+model of my system. I take the red and green and blue pieces and arrange them together under the
+title "Gate", and now if there's an error, I can tell where it belongs.
+
+It goes back to the *space and time* analogy. The operational experience of Wing Console is based on
+Wing's *spacial view* of the system. Every resource or event can be *traced back* to the logical
+unit it belongs to, based on your source code. This is basically the purpose of stack traces in
```

Comment:
Might need to add the space and time analogy back or reword this

Reply:
fuck

Reply:
reworded